src/reference/informative/informC08_interpretation_of_mathematics2.rst

```diff
@@ -0,0 +1,216 @@
+.. _informC08_interpretation_of_mathematics2:
+
+.. container:: toggle
+
+  .. container:: header
+
+    See more
+
+  .. container:: infospec
+
+    .. container:: heading3
+
+      Understanding how the mathematics works
+
+    The mathematics of a mathematical model is a collection of statements which are held to be true at all times.
+    These are distinct from initial conditions of a model (specified using the :code:`initial_value` attribute on :code:`variable` elements) which are only true initially.
+    You can read more about initialisation on the :ref:`Interpretation of initial_values<>` page and in the section below this one.
+
+    The collection of mathematical statements in CellML are the set of top-level children of a :code:`math` elements inside "pertinent components" (as explained in the previous block).
+    The example from the previous point is extended below in pseudocode, with the full CellML code beneath the link below.
+    The final output from the model - the total runs scored - is found by reducing all of the :code:`math` elements from all pertinent components in the model, whether or not those components interact.  In this example, the same mathematical situation could be written:
+
+    .. math::
+
+      r_{total} = r_{tom} + r_{dick} + r_{harry}
+
+      r_{total} = 99 + 0 + 10t
+
+    where :math:`r` is the number of runs, and :math:`t` is time.
+
+    .. code::
+
+      model: BackyardCricket
+        │
+        └─ component: FirstGame
+            │  ├─ variable: dicks_runs <╴╴╴╴╴╴┐
+            │  ├─ variable: harrys_runs <╴╴┐  ╷
+            │  ├─ variable: toms_runs <╴┐  ╷  ╷
+            │  ├─ variable: time        ╷  ╷  ╷
+            │  └─ math: total_runs = toms_runs + dicks_runs + harrys_runs
+            │                           ╵  ╵  ╵
+            ├─ component: Tom           ╵  ╵  ╵
+            │  ├─ variable: runs ╴╴╴╴╴╴╴┘  ╵  ╵
+            │  └─ math: runs = 99          ╵  ╵ connected
+            │                              ╵  ╵ variables
+            └─ component: Harry (imported) ╵  ╵
+                │  ├─ variable: runs ╴╴╴╴╴╴┘  ╵
+                │  ├─ variable: time          ╵
+                │  ├─ variable: dicks_runs ╴╴╴┤
```

Comment:
Why do you define `dicks_runs` if it is not used?

Reply:
Ok, it's to pass it up. Strange, but why not...

Reply:
It's the transition variable so that it can be passed up through the encapsulation hierarchy?

Reply:
How else would/should it be done?